crytic_compile/platform/etherscan.py

```diff
@@ -474,7 +474,9 @@ def _convert_version(version: str) -> str:
     Returns:
         str: converted version
     """
-    return version[1 : version.find("+")]
+    if "+" in version:
+        return version[1 : version.find("+")]
+    return version
```

Comment:
Do versions with no "+" also have no "v" in the beginning? With this change, versions with "+" won't have the first character removed.

Reply:
This was not really necessary (as the code worked with the `v`), but I pushed a quick fix for it anyway.